### PR TITLE
Python txamqp

### PR DESCRIPTION
**Pull request by hagna**
Maintainer: @hagna
Compile tested: x86, generic, latest
Run tested: x86, generic, latest

Description:

Python library for communicating with AMQP peers and brokers using Twisted

### CODE REVIEW

File: lang/python-txamqp/Makefile [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#
+# Copyright (C) 2009-2015 OpenWrt.org
```
Comment (yousong) [inline lang/python-txamqp/Makefile]: It was only recently said that you use this copyright header unless there was a formal legal document transferring copyright from you to OpenWrt.  If that is not the case, I would recommend using your own name in the header.

File: lang/python-txamqp/Makefile [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+include $(INCLUDE_DIR)/package.mk
+$(call include_mk, python-package.mk)
+
+define Package/python-txamqp
+	SECTION:=lang-python
+	CATEGORY:=Languages
+	SUBMENU:=Python
+	TITLE:=python-txAMQP
+	URL:=https://launchpad.net/txamqp
+	DEPENDS:=+python +libgmp +twisted
```
Comment (yousong) [inline lang/python-txamqp/Makefile]: txAMQP itself itself should be a pure python library and libgmp not part the dependency.

File: lang/python-txamqp/Makefile [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=txAMQP
+PKG_VERSION:=0.6.2
+PKG_RELEASE:=1
+PKG_MAINTAINER:=Nate <natester@gmail.com>
```
Comment (yousong) [inline lang/python-txamqp/Makefile]: Please consider using your full name in this field to make this more complete and formal.

**Comment by champtar** [comment]
Hi @hagna,
Closing for now, don't hesitate to reopen and have a look at @yousong suggestions
also you need to signoff your commits

**Pull request closed without merging**